discord/interactions/connectCommand.ts

```diff
@@ -169,14 +206,19 @@ export const handleConnectCommand = async (
         otherNetwork(alreadyDiscordMembers[0].starknetNetwork) ?? "";
       await DiscordMemberRepository.save(newDiscordMember);
 
+      //verification process based on network type
+      const verificationUrl = isEthereumNetwork(
```

Comment:
You could add a "verifyPage" field to network.json.
So that here you could do something like 
```ts
      const verificationUrl = `${config.BASE_URL}/${network.verifyPage}/${guildId}/${userId}/${newDiscordMember.customLink}`
```
Which is cleaner imo. As you copied this in several places in this file, this would be a cool improvement. You can also wrap it in a function if it's cleaner too